DepositeCalcTests/Tests/HistoryPageTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using NUnit.Framework;
+using DepositeCalcTests.Pages;
+using System;
+using System.Threading;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class HistoryPageTests : BaseTest
+    {
+        private HistoryPage historyPage;
+
+        [SetUp]
+        public void Setup()
+        {
+            InitDriver("https://localhost:5001/History");
+            historyPage = new HistoryPage(driver);
+        }
+
+        [Test]
+        public void CalculatorLinkTest()
+        {
+            // Act
+            var calculatorPage = historyPage.Calculator();
+
+            // Assert
+            Assert.IsTrue(calculatorPage.IsOpened(), "Incorrect page");
+        }
+
+        [Test]
+        public void TitleTest()
+        {
+            // Assert
+            Assert.AreEqual("History", driver.Title, "Incorrect title");
+        }
+
+        [Test]
+        public void LastCalculationHistoryTest()
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var depositAmount = "123";
+            var interestRate = "100";
+            var investmentTerm = "321";
+            var expectedLastCalculationValues = new[]
+            {
+                "123.00",
+                "100",
+                "321",
+                "365",
+                "231.17",
+                "108.17",
+            };
+
+            // Act
+            driver.Url = "https://localhost:5001/Calculator";
+            calculatorPage.ClearingMandatoryTextFields();
+            calculatorPage.FinancialYear = "365";
+            calculatorPage.FillingMandatoryTextFields(depositAmount, interestRate, investmentTerm);
```

Comment:
1. Let's create an Open() method and move `driver.Url = "https://localhost:5001/Calculator";` to it. So the test will look much better.
2. I think you don't need to 'clear fields' here.
```suggestion
            calculatorPage.Open();
            calculatorPage.FinancialYear = "365";
            calculatorPage.FillingMandatoryTextFields(depositAmount, interestRate, investmentTerm);
```